layouts/shortcodes/config-uri-existing-arrikto.html

```diff
@@ -1 +1 @@
-https://raw.githubusercontent.com/kubeflow/manifests/dc04ff600cee722d93cf80d413aa73ddd8387f1f/kfdef/kfctl_existing_arrikto.0.7.0.yaml
\ No newline at end of file
+https://raw.githubusercontent.com/kubeflow/manifests/b37bad9eded2c47c54ce1150eb9e6edbfb47ceda/kfdef/kfctl_existing_arrikto.0.7.1.yaml
```

Comment:
Remove the empty line at the end of the file. (We use the shortcode as a global variable, and if you add a line break, that line break is included wherever the variable is used.)

Reply:
@sarahmaddox it seems this is the result of using vim, thanks for catching that.